EvidenceApi.Tests/V1/UseCase/FindDocumentSubmissionByIdUseCaseTests.cs

```diff
@@ -15,48 +17,87 @@ public class FindDocumentSubmissionByIdUseCaseTests
         private FindDocumentSubmissionByIdUseCase _classUnderTest;
         private Mock<IEvidenceGateway> _evidenceGateway;
         private Mock<IDocumentTypeGateway> _documentTypesGateway;
+        private Mock<IDocumentsApiGateway> _documentsApiGateway;
         private readonly IFixture _fixture = new Fixture();
 
         private DocumentType _documentType;
         private DocumentSubmission _found;
+        private DocumentSubmission _found2;
+        private Task<Claim> _claim;
```

Comment:
Small thing but you could name this `_claim1` to be consistent with the other field names